test/box/gh-5518-add-granularity-option.test.lua

```diff
@@ -0,0 +1,47 @@
+
+-- write data recover from latest snapshot
+env = require('test_run')
+test_run = env.new()
+
+test_run:cmd('create server test with script="box/gh-5518-add-granularity-option.lua"')
+-- Start server test with granularity = 1 (rounded up to 4)
+test_run:cmd('start server test with args="1"')
+test_run:grep_log("test", "Changed to 4")
+test_run:cmd('switch test')
+s = box.schema.space.create('test')
+_ = s:create_index('test')
+-- Try to insert/delete to space, in case when UB sanitizer on,
+-- we check correct memory aligment
+for i = 1, 1000 do s:insert{i, i + 1} end
+for i = 1, 1000 do s:delete{i} end
+s:drop()
```

Comment:
Could you check memory consumption to make sure that less/more memory is used with different alignments?